components/engine/vendor.conf

```diff
@@ -104,15 +104,15 @@ github.com/googleapis/gax-go da06d194a00e19ce00d9011a13931c3f6f6887c7
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 
 # containerd
-github.com/containerd/containerd 89623f28b87a6004d4b785663257362d1658a729 # v1.0.0
+github.com/containerd/containerd cfd04396dc68220d1cecbe686a6cc3aa5ce3667c # v1.0.2
 github.com/containerd/fifo fbfb6a11ec671efbe94ad1c12c2e98773f19e1e6
-github.com/containerd/continuity 35d55c5e8dd23b32037d56cf97174aff3efdfa83
-github.com/containerd/cgroups 29da22c6171a4316169f9205ab6c49f59b5b852f
+github.com/containerd/continuity cf279e6ac893682272b4479d4c67fd3abf878b4e
```

Comment:
this one was already 4 commits ahead; https://github.com/containerd/continuity/compare/cf279e6ac893682272b4479d4c67fd3abf878b4e...35d55c5e8dd23b32037d56cf97174aff3efdfa83

Reply:
I removed this change.